pymobiledevice3/cli/developer.py

```diff
@@ -97,6 +99,21 @@ def applist(lockdown, color):
         print_json(apps, colored=color)
 
 
+@dvt.command('signal', cls=Command)
+@click.argument('pid', type=click.INT)
+@click.argument('sig', type=click.INT, required=False)
+@click.option('-s', '--signal-name', type=click.Choice([s.name for s in signal.Signals]))
+def send_signal(lockdown, pid, sig, signal_name):
+    """ Send SIGNAL to process by its PID """
+    if not sig and not signal_name:
+        raise MissingParameter(param_type="argument|option", param_hint="\"SIG|SIGNAL-NAME\"")
+    if sig and signal_name:
+        raise UsageError(message="Cannot give SIG and SIGNAL-NAME together")
```

Comment:
Use `'` instead `"`

Reply:
Sure

Reply:
still unchanged

Reply:
answered from my phone to confirm. now it's fixed :)